plugin/trino-apache-ranger/src/main/java/io/trino/plugin/ranger/RangerConfig.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ranger;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+import java.util.List;
+
+public class RangerConfig
+{
+    public static final String RANGER_TRINO_DEFAULT_SERVICE_NAME = "dev_trino";
+
+    private String serviceName = RANGER_TRINO_DEFAULT_SERVICE_NAME;
```

Comment:
I don't think we should hard code `dev_trino` as default service name. I would prefer to be required configuration required that it has to be provided during the setup.